apps/root/src/state/balances/balancesUpdater.tsx

```diff
@@ -0,0 +1,19 @@
+// import { IntervalSetActions } from '@constants/timing';
+// import { useAppDispatch } from '@hooks/state';
+// import useInterval from '@hooks/useInterval';
+// import { fetchInitialBalances, fetchPricesForAllChains } from './actions';
+
+const BalancesUpdater = () => {
```

Comment:
We can discuss about using the api for updating balances, otherwise I can continue this in a new task 😄

Reply:
Done